test/Conversion/FIRRTLToHW/lower-to-hw.mlir

```diff
@@ -1,7 +1,7 @@
-// RUN: circt-opt -lower-firrtl-to-hw -verify-diagnostics %s | FileCheck %s
-// RUN: circt-opt -pass-pipeline="lower-firrtl-to-hw{disable-mem-randomization}" -verify-diagnostics %s | FileCheck %s --check-prefix DISABLE_RANDOM --implicit-check-not RANDOMIZE_MEM
-// RUN: circt-opt -pass-pipeline="lower-firrtl-to-hw{disable-reg-randomization}" -verify-diagnostics %s | FileCheck %s --check-prefix DISABLE_RANDOM --implicit-check-not RANDOMIZE_REG
-// RUN: circt-opt -pass-pipeline="lower-firrtl-to-hw{disable-mem-randomization disable-reg-randomization}" -verify-diagnostics %s | FileCheck %s --check-prefix DISABLE_RANDOM --implicit-check-not RANDOMIZE_MEM --implicit-check-not RANDOMIZE_REG
+// RUN: circt-opt -pass-pipeline="builtin.module(lower-firrtl-to-hw)" -verify-diagnostics %s | FileCheck %s
```

Comment:
At some point I'd like to go through and remove the `-verify-diagnostics` part of invocations in the tests that don't need/use it, don't think it's a problem as much as unnecessary and possibly misleading.  I'm not certain but believe only the `*-errors.*` tests need these.  Anyway, not related to bump... :smile:.